mrmustard/math/tensorflow.py

```diff
@@ -374,6 +375,7 @@ def eigh(tensor: tf.Tensor) -> Tensor:
         """Returns the eigenvalues and eigenvectors of a matrix."""
         return tf.linalg.eigh(tensor)
 
+    # pylint: disable=arguments-differ
```

Comment:
(minor) would it perhaps be better to add the `rtol` and `atol` arguments to `MathInterface.sqrtm` (and also `TFMath.sqrtm`)?

Reply:
I guess that, as Filippo mentioned before, no need to clutter the interface with extra args that not all backends will implement. For example, torch backend does not use the `rtol` and `atol` args.